<ui version="4.0" >
 <author>(C) 2005 Andras Mantia &lt;amantia@kde.org></author>
 <comment>***************************************************************************
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; either version 2 of the License, or     *
 *   (at your option) any later version.                                   *
 *                                                                         *
 ***************************************************************************
</comment>
 <exportmacro></exportmacro>
 <class>SecondWizardPageBase</class>
 <widget class="QWidget" name="SecondWizardPageBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>620</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>32000</width>
    <height>32000</height>
   </size>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="5" row="0" column="0" >
    <widget class="QLabel" name="imagelabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>120</width>
       <height>1000</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QCheckBox" name="checkPrefix" >
     <property name="text" >
      <string>&amp;Use preview prefix</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLineEdit" name="linePrefix" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QGroupBox" name="GroupBox3" >
     <property name="title" >
      <string>Resource folders</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel1_2_3" >
        <property name="text" >
         <string>Templates:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="linePrjToolbar" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2_2" >
        <property name="text" >
         <string>Toolbars:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="linePrjTmpl" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="Q3ButtonGroup" name="bGroupSources" >
     <property name="title" >
      <string>Import Options</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="Line" name="line2" >
        <property name="frameShape" >
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QRadioButton" name="radioLocal" >
        <property name="text" >
         <string>&amp;Add local or remote files</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QRadioButton" name="radioWeb" >
        <property name="text" >
         <string>Use wget to &amp;download files from a site</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="checkInsertLocalTemplates" >
        <property name="text" >
         <string>Insert &amp;local templates</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkInsertGlobalTemplates" >
        <property name="text" >
         <string>Insert &amp;global templates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends></extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkPrefix</tabstop>
  <tabstop>linePrefix</tabstop>
  <tabstop>linePrjTmpl</tabstop>
  <tabstop>linePrjToolbar</tabstop>
  <tabstop>checkInsertGlobalTemplates</tabstop>
  <tabstop>checkInsertLocalTemplates</tabstop>
  <tabstop>radioLocal</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkPrefix</sender>
   <signal>toggled(bool)</signal>
   <receiver>linePrefix</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>249</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel" >
     <x>260</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPrefix</sender>
   <signal>toggled(bool)</signal>
   <receiver>TextLabel1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>187</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel" >
     <x>163</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
